### PR TITLE
build(other): add config for nvm

### PR DESCRIPTION
**Pull request by einhornimmond**
## 🍰 Pullrequest

- add .nvmrc file for every module containing same node version like in docker
- with installed nvm you need only to call `nvm use` in the module folder to use the correct NodeJS version
- you need to install yarn for every node version with `npm i -g yarn`

### CODE REVIEW

File: admin/.nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v14.17.0
```
Comment (clauspeterhuebner) [inline admin/.nvmrc:1]: Does it make sense to upgrade the version as soon as possible or why using such an old version?

File: backend/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v18.7.0
```
Comment (clauspeterhuebner) [inline backend/.nvmrc:1]: I know this modul will work with V20.0.0 too.
We should plan an upgrade for all moduls as soon as possible, right?

File: frontend/.nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+lts/gallium
```
Comment (clauspeterhuebner) [inline frontend/.nvmrc:1]: Is this a correct entry?
I expected a version-number instaed of a path-string...

**Review by clauspeterhuebner — APPROVED** [review]
Some questions and remarks possibly for further actions/tickets, so i approved it...

File: backend/.nvmrc (hunk 2 above)
Comment (einhornimmond) [reply backend/.nvmrc:1]: Yes right, but in another pr.

File: frontend/.nvmrc (hunk 3 above)
Comment (einhornimmond) [reply frontend/.nvmrc:1]: Like I write, I copy the version from docker. 
The Frontend Docker use node:16-alpine3.14 which is the latest version from 16 named gallium, so this string does the same. Tell nvm to use the latest release from gallium.

**Pull request merged**